.env.production

```
@@ -0,0 +1,47 @@
+# Production Environment Variables
+# Copy this to .env and fill in your values before deploying
+
+# App Configuration
+APP_DOMAIN=api.marketingtool.pro
+ENVIRONMENT=production
+
+# Database
+DB_USER=appwrite
+DB_PASSWORD=your_secure_db_password_here
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The DB_PASSWORD key should go before the DB_USER key

(UnorderedKey)